### PR TITLE
Bodies and spaces

### PR DESCRIPTION
**Pull request by jlazar17**
Merging master branch with bodies refactor

### CODE REVIEW

File: taurunner/body/body.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import sys
```
Comment (apizzuto) [inline taurunner/body/body.py:2]: Don't need sys anymore

File: taurunner/cross_sections/xs.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
+#cross section module
+import os
+import sys
+info = sys.version_info
+pyv  = int(info.major)
+import numpy as np
+import nuSQuIDS as nsq
```
Comment (apizzuto) [inline taurunner/cross_sections/xs.py:7]: Try except for nuSQUIDS

File: taurunner/cross_sections/xs.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+#import nuSQUIDSpy as nsq
+from python.modules import units
```
Comment (apizzuto) [inline taurunner/cross_sections/xs.py:9]: incorrect module name here

**Review by ibsafa — APPROVED** [review]
easy

**Pull request merged**